jetbrains-rider/tst/base/AwsBaseTestWithMarkup.kt

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package base
+
+import com.intellij.openapi.util.SystemInfo
+import com.jetbrains.rider.test.base.BaseTestWithMarkup
+import com.jetbrains.rider.test.base.PrepareTestEnvironment
+import com.jetbrains.rider.test.scriptingApi.setUpCustomToolset
+import com.jetbrains.rider.test.scriptingApi.setUpDotNetCoreCliPath
+import org.testng.annotations.BeforeClass
+
+// BaseTestWithMarkup inherit the logic in EditorTestBase class that prepare test project that create
+// an empty solution and adds project under test to this solution.
+//
+// When running with LOCAL_ENV_RUN flag set to true (for running tests outside of internal IntelliJ networks),
+// Rider auto-detect and use Rider's bundled MSBuild that might is not compatible with full .NET framework installed
```

Comment:
```suggestion
// Rider will auto-detect and use Rider's bundled MSBuild that might not be compatible with full .NET framework installed
```

Reply:
@justinmk3, sorry, I did not see the change before force-pushed the latest updated commit. Applied your change. My bad.